packages/manager/modules/vps/src/dashboard/advices/controller.js

```diff
@@ -0,0 +1,31 @@
+import get from 'lodash/get';
+
+export default class DedicatedServerVpsAdvicesCtrl {
+  /* @ngInject */
+  constructor($http) {
+    this.$http = $http;
+  }
+
+  $onInit() {
+    this.loading = true;
+    this.adviceGroups = [];
+    this.loadAdvices()
+      .then((adviceGroups) => {
+        this.adviceGroups = adviceGroups;
+      })
+      .finally(() => {
+        this.loading = false;
+      });
+  }
+
+  loadAdvices() {
+    return this.$http
+      .get(`/advices/vps/${this.serviceName}`, {
```

Comment:
Maybe we could have a common `ovh-api-advices` (or another name) that take two parameters : `serviceType` and `serviceName` and call the API `/advices/${this.serviceType}/${this.serviceName}` and use `ovh-advices` to display results ?

Reply:
I have moved logic of fetching all advices to module. All service has to do is to use the component and pass service name and service type.